### PR TITLE
Fix: init Loading State when promise is passed after click

### PR DESCRIPTION
**Pull request by maxkorz**
In pull request https://github.com/johannesjo/angular2-promise-buttons/pull/12 I broke something :-(
This PR fixes the bug and and I added a test for this use-case.

### CODE REVIEW

File: src/promise-btn.directive.ts [hunk 1]
```diff
@@ -200,7 +200,7 @@ export class PromiseBtnDirective implements OnDestroy, AfterContentInit {
   @HostListener('click')
   handleCurrentBtnOnly() {
     if (!this.cfg.handleCurrentBtnOnly) {
-      return false; // return false for testing
```
Comment (maxkorz) [inline src/promise-btn.directive.ts]: as it turns out, `return false` prevents Angular code from running

**Comment by maxkorz** [comment]
Don't merge yet. The fix is correct but the test is not working.

**Comment by maxkorz** [comment]
I couldn't fix the test completely. That's very strange.

When I do `$event.preventDefault();` in `handleCurrentBtnOnly`:
1. **I can't click** on the "Submit" button in the "Inside a form" section in the demo
2. **The test fails**

but, when I do `return false` in `handleCurrentBtnOnly`:
1. **I can't click** on the "Submit" button in the demo
2. **The test passes**

Sadly, I have no idea how to make the test work for the `return false` case.
If you have no idea how to make the test handle `return false` case, you can merge the PR, the issue is fixed

**Comment by johannesjo** [comment]
Thanks for the PR. I'll try to check everything again later this week.

**Pull request merged**